content/en/monitors/monitor_types/outlier.md

```diff
@@ -16,133 +16,89 @@ further_reading:
   text: "Consult your monitor status"
 ---
 
-Outlier Detection is an algorithmic feature that allows you to detect when some members of a group are behaving strangely compared to the others. For example, you could detect that one web server in a pool is processing an unusual number of requests, and hence should be a target for replacement. Or, you could get an early warning that significantly more 500s are happening in one AWS Availability Zone (AZ) than the others, which might indicate an issue brewing in that AZ.
+## Overview
 
-{{< img src="monitors/monitor_types/outliers/outliers-metric-alert.png" alt="outliers metric alert" responsive="true" style="width:80%;">}}
-
-## How to Use Outlier Detection on Your Data
-
-The `outliers` query function, when applied to your query, returns the usual results but with outliers series marked.
-
-You can use this function to display and alert on outliers in your data. To try it out, you'll first need a metric for which a group of hosts (or availability zones, partitions, etc) should exhibit uniform behavior. For the function to work, be sure that there are at least 3 or more members in the group. Given that, here are two ways to use outlier detection on that group.
-
-### Show Outliers in Dashboards or Screenboards
-
-Here's a graph of Gunicorn requests by host with outlier detection enabled.
-
-{{< img src="monitors/monitor_types/outliers/outliers-graph-dbscan-gunicorn.png" alt="outliers graph dbscan Gunicorn" responsive="true" style="width:80%;">}}
-
-You can see that one of the series is an outlier: it is handling significantly lower traffic than the others for the time window in question.
-
-To set up an outlier detection graph for your data add a metric to the graph showing all series in the groups. Then apply the outlier detection algorithm by adding the `outliers` function on your data. After applying the function, any outlier series is colored with a bold, warm palette, while all other series are colored with a lightweight, greyscale color palette.
-
-First create a new timeseries graph on your dashboard with your chosen metric.
-
-{{< img src="monitors/monitor_types/outliers/outliers-dash-choose-metrics-updated.png" alt="outliers dash choose metrics updated" responsive="true" style="width:80%;">}}
-
-To enable outlier detection, click on the `+` icon on the right side of the metrics line. Choose **Algorithms** from the function categories, then pick one of the four outlier algorithms.
-
-{{< img src="monitors/monitor_types/outliers/outliers-algorithm-selector.png" alt="outliers algorithm selector" responsive="true" style="width:80%;">}}
+Outlier Detection is an algorithmic feature that allows you to detect when some members of a group are behaving different compared to the others. For example, you could detect that one web server in a pool is processing an unusual number of requests, or significantly more 500 errors are happening in one AWS availability zone than the others.
```

Comment:
```suggestion
Outlier Detection is an algorithmic feature that allows you to detect when some tag values associated with a tag key are behaving different compared to the others. For example, you could detect that one web server in a pool is processing an unusual number of requests, or significantly more 500 errors are happening in one AWS availability zone than the others.
```

Reply:
This terminology makes sense to us but using it in the overview section could cause confusion.

Reply:
🤔 it's a tricky spot, should we describe technically what is happening ? or should we use concepts to describe the behaviour?

Reply:
If it's helpful, I've used the language "when a group is behaving differently than its peers".